docs/docs/administration/database-info.md

```diff
@@ -0,0 +1,49 @@
+| typeorm_metadata          |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| user_token                | Contains information about the devices connected to Immich, device ID, token, creation and modification date, user ID, device type, operating system type.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| users                     | Contains information about the users of the system, email, password (encrypted), creation and update date, path to profile picture (optional), whether system administrator?, need to change password?, deletion date, oauthId (optional), storage label name (optional), whether Memories enabled?, username, avatar color (optional), available storage quota (optional), storage quota in use.                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
I probably wouldn't try to give an exhaustive list here. If you want to include something like this I think it would be nice to briefly explain the *most important* tables. Something like `typeorm_metadata` or others however don't matter at all to get a quick overview.